Makefile

```diff
@@ -0,0 +1,25 @@
+##############################################################
+#               CMake Project Wrapper Makefile               #
+############################################################## 
+
+SHELL := /bin/bash
+RM    := rm -rf
+
+all: ./build/Makefile
+	@ $(MAKE) -C build
+
+./build/Makefile:
+	@ (cd build >/dev/null 2>&1 && cmake ..)
+
+clean:
+	@- (cd build >/dev/null 2>&1 && cmake .. >/dev/null 2>&1)
+	@- $(MAKE) --silent -C build clean || true
+	@- $(RM) ./build/Makefile
```

Comment:
That's related to Issue [#11] ( https://github.com/LeMonADE-project/LeMonADE/issues/11 ) Theres the need for a folder for the build process. The installed lib and executables maybe also placed there if we not specify a installation directory by -DINSTALLDIR_LEMONADE. I'm open for suggestions.